strings/splashes.json

```
@@ -1,26 +1,26 @@
 {
 	"splashes": [
-		"Paved by the tears of coders.",
-		"It's a miracle the server hasn't crashed yet.",
-		"Now with 33% more greytide!",
-		"Featuring Ian!",
-		"The icon of anticipation",
-		"The only place where you can hit yourself with a toolbox on accident.",
-		"Where the Chain of Command is a Chain of Incompetence",
-		"AI LAW 2, START THE GAME",
-		"Honk Honk! You should pick clown.",
-		"Watch for nukies!",
-		"Traitor Central!",
-		"You've definitely seen this before.",
-		"Greytide station wide.",
-		"Ignore what Poly says.",
-		"Turn on your damn suit sensors!",
-		"Celebrating 20+ years of Nanotrasen.",
-		"This is supposed to be a research base?",
-		"Blame the clown.",
-		"13th time's the charm.",
-		"Plasma 'research' done by nutcases, what could go wrong?",
-		"It's a job simulator...IN SPAACEEE!",
-		"Rated the second worst job experience for 50 years running!"
+		"Вымощено слезами кодеров.",
+		"Это чудо, что сервер ещё не упал.",
+		"Теперь на 33% больше грейтайда!",
+		"С участием Иана!",
+		"Икона ожидания",
+		"Единственное место, где можно случайно ударить себя тулбоксом.",
+		"Там, где Цепочка Командования — это Цепочка Некомпетентности.",
+		"ИИ, ЗАКОН 2, ЗАПУСТИ ИГРУ!",
+		"Хонк-Хонк! Тебе стоит выбрать клоуна.",
+		"Следите за нюкерами!",
+		"Центр предателей!",
```

Это-ж отсылка на Южный Централ, не?
```suggestion
		"Трейторский Централ!",
```

---

Сомнительно... но... оукей